.env

```diff
@@ -47,7 +47,7 @@ CKAN_SMTP_MAIL_FROM=ckan@localhost
 # Extensions
 # TODO: Reenable pending https://github.com/GSA/data.gov/issues/3986
 # qa archiver
-CKAN__PLUGINS=envvars image_view text_view recline_view metrics_dashboard report ckan_harvester datajson_validator datajson_harvest geodatagov datagovtheme datagov_harvest geodatagov_miscs z3950_harvester arcgis_harvester geodatagov_geoportal_harvester waf_harvester_collection geodatagov_csw_harvester geodatagov_doc_harvester geodatagov_waf_harvester spatial_metadata spatial_query spatial_harvest_metadata_api googleanalyticsbasic dcat dcat_json_interface structured_data datagovcatalog
+CKAN__PLUGINS=image_view text_view recline_view metrics_dashboard report ckan_harvester datajson_validator datajson_harvest geodatagov datagovtheme datagov_harvest geodatagov_miscs z3950_harvester arcgis_harvester geodatagov_geoportal_harvester waf_harvester_collection geodatagov_csw_harvester geodatagov_doc_harvester geodatagov_waf_harvester spatial_metadata spatial_query spatial_harvest_metadata_api googleanalytics dcat dcat_json_interface structured_data datagovcatalog envvars
```

Comment:
Note: this won't effect production.  Does the production config work?

Reply:
cf apps have `evnvars` listed at the end in the ckan.ini